enigma-swing/src/main/java/cuchaz/enigma/gui/Gui.java

```diff
@@ -104,18 +111,34 @@ public class Gui {
 	public SearchDialog searchDialog;
 
 	public Gui(EnigmaProfile profile, Set<EditableType> editableTypes) {
+		// right panels
+		// top panels
+		RightPanel.registerPanel(new CallsTree(this));
+		RightPanel.registerPanel(new ImplementationsTree(this));
+		RightPanel.registerPanel(new InheritanceTree(this));
+		RightPanel.registerPanel(new StructurePanel(this));
+
+		// bottom panels
+		RightPanel.registerPanel(new MessagesPanel(this));
+		RightPanel.registerPanel(new UsersPanel(this));
```

Comment:
Might be worth merging these two into a "Online" tab or similar, just one of the two isn't going to use the whole side on its own